docs/api/v1.24.md

```diff
@@ -2648,6 +2648,7 @@ Docker daemon report the following event:
 **Status codes**:
 
 -   **200** – no error
+-   **400** - bad parameter
```

Comment:
how about use `bad request` here because it's define in https://tools.ietf.org/html/rfc7231#section-6.5.1 ? @thaJeztah  WDYT?